helm/examples/azure/log-analytics/loganalytics-hpcc-logaccessV2.yaml

```diff
@@ -0,0 +1,94 @@
+# Configures HPCC logAccess to target Azure Log Analytics Workspace
+global:
+  logAccess:
+    name: "Azure LogAnalytics LogAccess"
+    type: "AzureLogAnalyticsCurl"
+    #connection:
+      #All connection attributes are optional.
+      #It is preferable to provide connection values as secret values category 'esp', secret name 'azure_logaccess'
+      # NOTE: secret 'azure_logaccess' must include 'aad-client-secret' and it cannot be provided in configuration
+      #
+      #workspaceID: "XYZ"      #ID of the Azure LogAnalytics workspace to query logs from
+                               # Secret value equivalent: 'ala-workspace-id'
+      #clientID: "DEF"         #ID of Azure Active Directory registered application with api.loganalytics.io access - format: 00000000-0000-0000-0000-000000000000
+                               # Secret value equivalent: 'aad-client-id'
+      #tenantID: "ABC"         #The Azure Active Directory Tenant ID, required for KQL API access
+                               # Secret value equivalent: 'aad-tenant-id'
+    logMaps:
+    - type: "global"
+      storeName: "ContainerLogV2"
+      searchColumn: "LogMessage"
+      timeStampColumn: "hpcc_log_timestamp"
+      columnType: "dynamic"
+      columnMode: "ALL"
+    - type: "workunits"
+      storeName: "ContainerLogV2"
+      searchColumn: "hpcc_log_jobid"
+      columnMode: "DEFAULT"
+      columnType: "string"
+    - type: "components"
+      storeName: "ContainerLogV2"
+      searchColumn: "ContainerName"
+      columnMode: "DEFAULT"
+      columnType: "string"
+      disableJoins: true
```

Comment:
If I understand correctly, this option was am option for V1 to potentially curtail expense of the joins necessary to get to the container names.
It doesn't make sense to be off (for that purpose at least), and the V2 implementation hard-wires it off afaics - so I think it should be removed from this V2 configuration file.